security-framework/src/item.rs

```diff
@@ -316,6 +324,13 @@ impl ItemSearchOptions {
                 ));
             }
 
+            if let Some(ref trusted_only) = self.trusted_only {
+                params.push((
+                    CFString::wrap_under_get_rule(kSecMatchTrustedOnly),
+                    if *trusted_only { CFBoolean::true_value().into_CFType() } else { CFBoolean::true_value().into_CFType() },
```

Comment:
Both branches are true

Reply:
Wouldn't want some sneaky untrusted certs getting out. Belt and suspenders? (...oops!) Fixup'd.